서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,6 @@
+function solution(seoul) {
+    
+    return '김서방은 ' + seoul.indexOf('Kim') + '에 있다';
+}
+      
+console.log(solution(["song", "hyun","gyoung", "Kim"]));
```

Comment:
문제 푸실  때  console.log로도 확인이 가능하지만 함수를 선언해 주는 자체가 return으로도 결과값을 도출해 낼 수 있기 때문에 굳이 console.log로 마무리 해주지 않아도 돼요

추가로 seoul.indexOf('Kim')이라는 메서드를 return할 때 선언해 주는 것 보다 변수로 먼저 선언해 준 다음 변수로 가져와 return하는게 효율성 면에서 더 좋기 때문에 앞으로는 변수로 먼저 선언 해주세요~

```suggestion
function solution(seoul) {
    let answer = seoul.indexOf('Kim');
    return '김서방은 ' + answer + '에 있다';
}
```